spec/app/models/mdm/service_spec.rb

```diff
@@ -1,5 +1,6 @@
 RSpec.describe Mdm::Service, type: :model do
   it_should_behave_like 'Metasploit::Concern.run'
+  include_context 'Rex::Text'
```

Comment:
If only some of the examples in this spec need `'Rex::Text'`, then the `include_context 'Rex::Text'` should be moved to a sub `context` that only affects the examples that need it.